2023-10-31 16:24:57,252 INFO    MainThread:346628 [wandb_setup.py:_flush():76] Loading settings from /home/tbertrand/.config/wandb/settings
2023-10-31 16:24:57,252 INFO    MainThread:346628 [wandb_setup.py:_flush():76] Loading settings from /home/tbertrand/Bureau/ULM_4channel_pull/wandb/settings
2023-10-31 16:24:57,252 INFO    MainThread:346628 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-10-31 16:24:57,252 INFO    MainThread:346628 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'figures_IOSTAR.py', 'program': './figures_IOSTAR.py'}
2023-10-31 16:24:57,252 INFO    MainThread:346628 [wandb_init.py:_log_setup():428] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20231031_162457-tlu4dgez/logs/debug.log
2023-10-31 16:24:57,253 INFO    MainThread:346628 [wandb_init.py:_log_setup():429] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20231031_162457-tlu4dgez/logs/debug-internal.log
2023-10-31 16:24:57,253 INFO    MainThread:346628 [wandb_init.py:init():462] calling init triggers
2023-10-31 16:24:57,253 INFO    MainThread:346628 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'nb': 20, 'threshold_landmarks': 0.4, 'threshold_metric': 0.2, 'threshold_tree': 0.05, 'xi': 1.0, 'eps': 1.0, 'power': 2.0, 'bool_erase_far': 1, 'dist': 0.01, 'decalage': 0, 'beta': 0.1}
2023-10-31 16:24:57,253 INFO    MainThread:346628 [wandb_init.py:init():515] starting backend
2023-10-31 16:24:57,253 INFO    MainThread:346628 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-10-31 16:24:57,261 INFO    MainThread:346628 [backend.py:ensure_launched():217] starting backend process...
2023-10-31 16:24:57,266 INFO    MainThread:346628 [backend.py:ensure_launched():222] started backend process with pid: 346694
2023-10-31 16:24:57,267 INFO    MainThread:346628 [wandb_init.py:init():525] backend started and connected
2023-10-31 16:24:57,272 INFO    MainThread:346628 [wandb_init.py:init():596] updated telemetry
2023-10-31 16:24:57,287 INFO    MainThread:346628 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2023-10-31 16:25:27,288 INFO    MainThread:346628 [interface_shared.py:_communicate_run():326] couldn't get run from backend
2023-10-31 16:25:27,288 ERROR   MainThread:346628 [wandb_init.py:init():636] backend process timed out
2023-10-31 16:25:27,288 ERROR   MainThread:346628 [wandb_init.py:init():647] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2023-10-31 16:25:27,294 ERROR   MainThread:346628 [wandb_init.py:init():1026] error
Traceback (most recent call last):
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 999, in init
    run = wi.init()
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 651, in init
    backend.cleanup()
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/backend/backend.py", line 246, in cleanup
    self.interface.join()
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 475, in join
    super().join()
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 666, in join
    _ = self._communicate_shutdown()
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 472, in _communicate_shutdown
    _ = self._communicate(record)
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/tbertrand/anaconda3/envs/ULM_landmark/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 231, in _communicate_async
    raise Exception("The wandb backend process has shutdown")
Exception: The wandb backend process has shutdown
